btcstaking-tracker/stakingeventwatcher/tracked_delegations.go

```diff
@@ -105,6 +105,28 @@ func (td *TrackedDelegations) AddDelegation(
 	return delegation, nil
 }
 
+func (td *TrackedDelegations) AddDel(
```

Comment:
hmm we now have to function to add to tracker `AddDelegation` and `AddDel` ? How should the caller know which function to use ? It would be great to:
- either have some docs about when each methods should be used
- or have only one methods to add to trackers

Reply:
Good point, not sure where my head was, removed the AddDel, it's completely unnecessary